docs/ldml/tr35-info.md

```diff
@@ -937,6 +962,8 @@ However, in conversion, it may be necessary to resolve them in order to find a m
 <!ATTLIST unitQuantity quantity NMTOKENS #REQUIRED >
 
 <!ATTLIST unitQuantity status NMTOKEN #IMPLIED >
+
+<!ATTLIST unitQuantity description CDATA #IMPLIED >
```

Comment:
This one isn't documented specifically; do you consider it covered by the docs for description above?

Reply:
Yes, I thought it was sufficient to description @description as a common attribute for these items.

Reply:
BTW, let me know if you'd still approve after seeing my replies.